docs/index.rst

```diff
@@ -1,62 +1,34 @@
 
-Welcome to Rikai's documentation!
-=================================
-
-Rikai is an `Apache Spark`_ based ML data format built for working with
-unstructured data at scale. Processing large amounts of data for ML is never trivial, but that
-is especially true for images and videos often at the core of deep learning applications. We are
-building Rikai with two main goals:
-
-1. Enable ML engineers/researchers to have a seamless workflow from feature engineering (`Spark`_)
-   to training (`PyTorch`_/`Tensorflow`_), from notebook to production (:doc:`See details <types>`).
-2. Enable advanced analytics capabilities to support much faster active learning, model debugging,
-   and monitoring in production.
-
-.. code-block:: python
-
-  from pyspark.ml.linalg import DenseMetrix
-  from rikai.types import Image, Box2d
-  from rikai import numpy as np
-
-  df = spark.createDataFrame(
-      [{
-          "id": 1,
-          "mat": DenseMatrix(2, 2, range(4)),
-          "image": Image("s3://foo/bar/1.png"),
-          "annotations": [
-              {
-                  "label": "cat",
-                  "mask": np.random(size=(256,256)),
-                  "bbox": Box2d(xmin=1.0, ymin=2.0, xmax=3.0, ymax=4.0)
-              }
-          ]
-      }]
-  )
-
-  df.write.format("rikai").save("dataset/out")
-
-``Rikai`` dataset can be seamlessly integrated into your favorite training frameworks,
-taking `Pytorch`_ as an example:
-
-.. code-block:: python
-
-  from rikai.torch import DataLoader
-
-  data_loader = DataLoader(
-      "dataset/out",
-      shuffle=True,
-      batch=8,
-  )
-  for examples in data_loader:
-      print(example)
-
-Additionally, the parquet-native nature of the ``rikai`` format allows such unstructured ML dataset
-being analyzed in `Jupyter`_, `Spark`_, `Presto`_ or
-`BigQuery <https://cloud.google.com/bigquery/external-data-cloud-storage>`_.
+Welcome to Rikai!
+=================
+
+What is Rikai
+-------------
+
+The success of a AI product is determined by high quality of ML data.
```

Comment:
The quality of an AI dataset can make or break an AI project, but tooling for AI data is sorely lacking in ergonomics. As a result, practitioners must spend most of their time and effort wrestling with their data instead of innovating on the models and use cases.